interop_binaries/src/testcontainer.rs

```diff
@@ -30,6 +30,7 @@ impl Default for Client {
     fn default() -> Self {
         // One-time initialization step: load compiled image into docker, recording its image tag,
         // so that we can launch it later.
+        #[allow(clippy::const_is_empty)]
```

Comment:
nit: add comments describing why this whitelisting is necessary (I would have had to think about it for a few minutes w/o the PR description)